packages/@aws-cdk/aws-apigatewayv2/lib/websocket/authorizer.ts

```diff
@@ -12,6 +12,9 @@ import { IWebSocketRoute } from './route';
 export enum WebSocketAuthorizerType {
   /** Lambda Authorizer */
   LAMBDA = 'REQUEST',
+
+  /** IAM Authorizer */
+  IAM = 'AWS_IAM',
```

Comment:
what does this do? Can you provide a link to the docs that describe what this does?

Reply:
Thanks for reviewing. So this is covered in the docs [here](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-websocket-control-access-iam.html). 

It works the same way as REST or HTTP API's where you can make an endpoint (connect route for websocket) publicly inaccessible, and setup an IAM user, and allow access using signed URL's

The above doc links back to [this](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-control-access-using-iam-policies-to-invoke-api.html) section which explains the `AWS_IAM`